index.js

```diff
@@ -223,11 +223,12 @@
           var fn = this.__outsideClickHandler;
           if (typeof document !== 'undefined') {
             var events = this.props.eventTypes || DEFAULT_EVENTS;
+            var passive = !this.props.preventDefault;
```

Comment:
hm, i see `pasive` strongly tied to the `preventDefault` and it seemed reasonable to deduce this here instead of adding yet another aceepted prop

This is not caused by a `click` event but by a `touchstart` which we use here by default - https://www.chromestatus.com/features/5093566007214080 . I guess Chrome decided to treat those as passive by default as `preventDefault` on a touch event prevents scrolling. 

> clicks should absolutely be able to prevent default browser behaviour

That is still possible, just need do this by using `preventDefault` prop instead of calling it manually. OTOH - those seems a little bit iffy to me, what's the reasoning of having those props like `preventDefault` and `stopPropagation` instead of leting the user to handle this on his own? Removing them + adding a `passive` prop would simplify the codebase I think.